aten/src/ATen/core/jit_type.h

```diff
@@ -1881,8 +1881,8 @@ struct TORCH_API ClassType : public NamedType {
   // getter and (optional) setter for that attribute.
   struct Property {
     std::string name;
-    const torch::jit::Function* getter;
```

Comment:
I am sure this is necessary change, but I didn't find code that modifies getter/setter. Could you point me to them?

Reply:
These are used (https://github.com/pytorch/pytorch/pull/50670/files#diff-c699f539322d99f6ab85853a11d9b7339acb49639d8797934c8fec2faa5ca7d4R109 AND https://github.com/pytorch/pytorch/pull/50670/files#diff-c699f539322d99f6ab85853a11d9b7339acb49639d8797934c8fec2faa5ca7d4R112)